pub_sub/csharp/checkout/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Dapr.Client;
+using System.Threading;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+
+while(true) {
+    Random random = new Random();
+    var order = new Order(random.Next(1,1000));
+    //var data = JsonSerializer.Serialize<Order>(order);
+    CancellationTokenSource source = new CancellationTokenSource();
+    CancellationToken cancellationToken = source.Token;
+    using var client = new DaprClientBuilder().Build();
+    await client.PublishEventAsync("order_pub_sub", "orders", order, cancellationToken);
+    //await client.PublishEventAsync("order_pub_sub", "orders", data, cancellationToken);
+    Console.WriteLine("Published data: " + order);
+    System.Threading.Thread.Sleep(1000);
+}
+
+public record Order([property: JsonPropertyName("orderid")] int order_id);

```

Comment:
Is there a reason to avoid camelCase (`orderId`). It's wierd to see this. Also this should be `int OrderId`. PascalCase for properties (yes this is a property) and camelCase for JSON.

Reply:
we will snap to this.